src/Features/CSharpTest/Snippets/AbstractCSharpAutoPropertySnippetProviderTests.cs

```diff
@@ -56,11 +56,14 @@ class MyClass
             """);
     }
 
-    [Fact]
-    public async Task InsertSnippetInRecordTest()
+    [Theory]
+    [InlineData("record")]
+    [InlineData("record struct")]
+    [InlineData("record class")]
+    public async Task InsertSnippetInRecordTest(string text)
```

Comment:
Please some more descriptive name for the parameter. I suggest `recordType`, but if you come up with something better, use that instead

Reply:
Renamed the param name